From 6b2061bc0699202f81aeb782f301f1bba9f8a826 Mon Sep 17 00:00:00 2001
From: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
Date: Mon, 30 Jan 2023 19:58:13 +0100
Subject: [PATCH] ichspi.c: effectively disable prettyprint_ich9_reg_hsfc()

[WORKAROUND]

Due to the fact that --progress is broken, heads relies on debug output
to track the progress.

This call makes the debug output so much cluttered.

See:
https://github.com/Dasharo/flashrom/pull/11#issuecomment-1409142374

Change-Id: Icd0f7de8a59382a4cf89afc63150ab1e7ca308dd
Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
---
 ichspi.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ichspi.c b/ichspi.c
index aefbe2f9de37..c94d9971fd0f 100644
--- a/ichspi.c
+++ b/ichspi.c
@@ -479,6 +479,7 @@ static void prettyprint_ich9_reg_hsfs(uint16_t reg_val, enum ich_chipset ich_gen
 
 static void prettyprint_ich9_reg_hsfc(uint16_t reg_val, enum ich_chipset ich_gen)
 {
+#if 0
 	msg_pdbg("HSFC: ");
 	pprint_reg(HSFC, FGO, reg_val, ", ");
 	switch (ich_gen) {
@@ -497,6 +498,7 @@ static void prettyprint_ich9_reg_hsfc(uint16_t reg_val, enum ich_chipset ich_gen
 	}
 	pprint_reg(HSFC, FDBC, reg_val, ", ");
 	pprint_reg(HSFC, SME, reg_val, "\n");
+#endif
 }
 
 static void prettyprint_ich9_reg_ssfs(uint32_t reg_val)
-- 
2.51.0

